steps/test/unit/tReorderMSData.cc

```diff
@@ -1,94 +1,115 @@
 
-#include <fstream>
 #include <algorithm>
+#include <fstream>
+#include <limits>
+
 #include <boost/test/unit_test.hpp>
 
 #include "tStepCommon.h"
-#include <dp3/steps/Step.h>
-#include "../../NullStep.h"
+#include "dp3/steps/Step.h"
 #include "../../InputStep.h"
+#include "../../NullStep.h"
 #include "../../ReorderMSData.h"
-#include <dp3/base/DPBuffer.h>
-#include <dp3/base/DPInfo.h>
 #include "../../../common/ParameterSet.h"
-#include "../../../common/StringTools.h"
 
-using dp3::base::DPBuffer;
-using dp3::base::DPInfo;
 using dp3::common::Fields;
 using dp3::common::ParameterSet;
 using dp3::steps::Step;
 
 BOOST_AUTO_TEST_SUITE(reorder)
 
-bool compareBinaries(const std::string& filename1,
-                     const std::string& filename2) {
-  std::ifstream file1(
-      filename1,
-      std::ifstream::ate | std::ifstream::binary);  // open file at the end
-  std::ifstream file2(
-      filename2,
-      std::ifstream::ate | std::ifstream::binary);  // open file at the end
+// Function to compare two binary files. If the files for comparison are Meta
+// file, the flag is_meta_file is set to true.
+void compareBinaryFiles(const std::string& reference_filename,
+                        const std::string& input_filename, bool is_meta_file) {
+  std::pair<std::istreambuf_iterator<char>, std::istreambuf_iterator<char>>
+      iterator_pair;
+
+  std::ifstream reference_file_ptr(reference_filename);
+  std::ifstream input_file_ptr(input_filename);
```

Comment:
Variable names now all seem correct in terms of casing, however for these two you've missed I think the multiple times I mentioned abbreviations in previous comments...
Generally according to the style guidelines I linked, we should avoid abbreviations (especially ones that are just leaving out letters) unless absolutely necessary.

In this case the `_ptr` abbreviation is completely unnecessary, especially because they are not actually pointers anyway, therefore it would be better to just remove this and name these `reference_file` and `input_file`

Reply:
Yes, dropeed the `_ptr` abbrevation.

Reply:
Not sure if you forgot to push a commit here, you say you removed the `_ptr` abbreviation but its still there

Reply:
Nevermind, the page was stale I see it is removed.